modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -424,12 +427,8 @@
      * @throws JMException Thrown if undeploy failed.
      */
     @MXBeanDescription("Makes the best attempt to undeploy a task from the whole grid.")
-    @MXBeanParametersNames(
-        "taskName"
-    )
-    @MXBeanParametersDescriptions(
-        "Name of the task to undeploy."
-    )
+    @MXBeanParametersNames("taskName")
```

Comment:
Lets do not change code that is not related to the issue

Reply:
Fixed